presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1726,4 +1727,17 @@ public HiveClientConfig setUseRecordPageSourceForCustomSplit(boolean useRecordPa
         this.useRecordPageSourceForCustomSplit = useRecordPageSourceForCustomSplit;
         return this;
     }
+
+    @Config("hive.hudi-metadata-enabled")
+    @ConfigDescription("For Hudi tables prefer to fetch the list of file names and sizes from metadata rather than storage")
```

Comment:
lets make this description more future proof. 

"For Hudi tables prefer to fetch the list of file names, sizes and other metadata from the internal metadata table rather than storage"